Dockerfile

```diff
@@ -1,27 +1,23 @@
-FROM alpine:3.6
+FROM alpine:3.7
 
-LABEL org.label-schema.name="docker-bench-security" \
-      org.label-schema.url="https://dockerbench.com" \
-      org.label-schema.vcs-url="https://github.com/docker/docker-bench-security.git"
+LABEL \
+  org.label-schema.name="docker-bench-security" \
+  org.label-schema.url="https://dockerbench.com" \
+  org.label-schema.vcs-url="https://github.com/docker/docker-bench-security.git"
 
-# Switch to the HTTPS endpoint for the apk repositories as per https://github.com/gliderlabs/docker-alpine/issues/184
-RUN sed -i 's/http\:\/\/dl-cdn.alpinelinux.org/https\:\/\/alpine.global.ssl.fastly.net/g' /etc/apk/repositories
+COPY ./*.sh /usr/local/bin/
+COPY ./tests/*.sh /usr/local/bin/tests/
 
+# Switch to the HTTPS endpoint for the apk repositories
+# https://github.com/gliderlabs/docker-alpine/issues/184
 RUN \
-  apk upgrade --no-cache && \
+  sed -i 's/http\:\/\/dl-cdn.alpinelinux.org/https\:\/\/alpine.global.ssl.fastly.net/g' /etc/apk/repositories && \
   apk add --no-cache \
     docker \
     dumb-init && \
-  rm -rf /usr/bin/docker-* /usr/bin/dockerd && \
-  mkdir /usr/local/bin/tests
-
-COPY ./*.sh /usr/local/bin/
-
-COPY ./tests/*.sh /usr/local/bin/tests/
+  rm -rf /usr/bin/docker?*
 
 WORKDIR /usr/local/bin
 
-HEALTHCHECK CMD exit 0
```

Comment:
This is https://github.com/docker/docker-bench-security/pull/220, I'll test if it's still required.

Reply:
```
[WARN] 4.6  - Ensure HEALTHCHECK instructions have been added to the container image
[WARN]      * No Healthcheck found: [docker-bench-security-ellerbrock:latest]
[WARN]      * No Healthcheck found: [alpine:3.7]
[WARN]      * No Healthcheck found: [alpine:3.6]
```